fala/apps/adviser/tests/test_cookies_page_journeys_playwright.py

```diff
@@ -31,8 +35,18 @@ def tear_down(self, page, context):
         page.context.close()
         context.browser.close()
 
+    def take_screenshot(self, page):
```

Comment:
Can you import this from the playwright setup python file? It's the same code.

Reply:
Not this particular method as we're not inheriting from this class.

We have a fresh set-up for cookies, as we want to keep the same context.